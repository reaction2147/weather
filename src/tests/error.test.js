import { render, screen, cleanup } from '@testing-library/react';
import Error from '../components/error';

afterEach(cleanup)

it("renders", () => {
  const {asFragment} = render(<Error />)
  expect(asFragment()).toMatchSnapshot()
})

it("has text", () => {
    const {getByTestId} = render(<Error />)
    expect(getByTestId("title")).toHaveTextContent("ERROR")
  })
  
  it("has text", () => {
    const {getByTestId} = render(<Error />)
    expect(getByTestId("sub_title")).toHaveTextContent("Please enter a valid location")
  })