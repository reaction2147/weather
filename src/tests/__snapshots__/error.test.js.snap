// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="bg-black/[.5] rounded shadow backdrop-blur-sm border-black/[.9] w-100"
  >
    <div
      class="flex flex-col justify-center p-4 text-red-700"
    >
      <h1
        class="text-2xl"
        data-testid="title"
      >
        ERROR
      </h1>
      <p
        class="text-xl"
        data-testid="sub_title"
      >
        Please enter a valid location
      </p>
    </div>
  </div>
</DocumentFragment>
`;
